assets/items/scripts/scanner_controls_option.gd

```diff
@@ -0,0 +1,3 @@
+class_name ScannerControlsOption extends CheckBox
```

Comment:
Is this necessary? Unless there are scene-specific changes to the checkbox, I feel like we can just store an array of checkboxes.